src/lap_stats/lap_stats.c

```diff
@@ -793,40 +793,33 @@ void lapstats_processUpdate(GpsSnapshot *gps_snapshot)
 
         /**
          * ============================================
-         * Set up interpolation for GPS Point
+         * Set up interpolation for GPS Points
          * ============================================
          */
+
+        /* Evenly split up difference in latitiude into intervals */
         float lat1 = gps_snapshot->previousPoint.latitude;
-        float lon1 = gps_snapshot->previousPoint.longitude;
         float lat2 = gps_snapshot->sample.point.latitude;
-        float lon2 = gps_snapshot->sample.point.longitude;
-
-        /**
-         * Evenly split up difference in latitiude into intervals
-         * longitude is linearly interpolated based on changing latitude
-         */
-        float lat_interval = fabsf(lat2 - lat1) / (float)interval_count;
-        if (lat1 > lat2)
-                /* Account for reverse direction */
-                lat_interval = -lat_interval;
-
+        float lat_interval = (lat2 - lat1) / (float)interval_count;
```

Comment:
Should we check to make sure `delta_since_last` or `interval_count` isn't zero to prevent div zero errors?

Reply:
The MAX in line 786 guards against 0 interval_count.  

Will look at delta_since_last